src/SFVBolivia/Helpers/Bill.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFVBolivia.Helpers
+{
+    public class Bill
+    {
+        private int billNumber;
```

Comment:
These private variables are not necessary, please remove it.  You can use the properties directly this.BillNumber = ...

Reply:
done.